app/controllers/find/courses_controller.rb

```diff
@@ -18,9 +18,20 @@ def show
         site_statuses: [:site]
       ).find_by!(course_code: params[:course_code]&.upcase).decorate
 
+      distance_from_location if params[:location]
+
       render_not_found unless @course.is_published?
     end
 
+    def distance_from_location
+      @coordinates = Geolocation::CoordinatesQuery.new(params[:location]).call
+      @distance_from_location ||= ::Courses::NearestSchoolQuery.new(
+        courses: [@course],
+        latitude: @coordinates[:latitude],
+        longitude: @coordinates[:longitude]
+      ).call.first.distance_to_search_location.ceil
```

Comment:
All published courses will have a school that returns first and distance_to_search_location right? Do we need to rescue or guard this?